### DISCUSSION TITLE
What's the thinking behind [Experimental] marked APIs if code/metadata is stripped?

### DISCUSSION THREAD

**Discussion by riverar** [microsoft/WindowsAppSDK#1801] [Q&A]
I'm a bit puzzled at the use of the metadata Experimental attribute on APIs when those APIs are stripped completely from the codebase, the metadata, and are uncallable.

**Comment by DrusTheAxe** [comment]
I think I know what you're asking but I'd like to be sure. Can you provide a concrete example? (this topic can get complicated and hard to follow in the abstract)
